src/InputWithLabel.js

```diff
@@ -0,0 +1,28 @@
+import React, { useEffect, useRef } from "react";
+
+function InputWithLabel({ handleTitleChange, todoTitle, children, isFocused }) {
+  const inputRef = useRef();
+
+  useEffect(() => {
```

Comment:
I like your use of the autoFocus attribute on the input! Using that, it actually works without the need for the hook - if you comment out that useEffect, the autofocus functionality still works. I prefer your solution, but just to follow the instructions, would you mind removing the autofocus attribute and get the auto focus functionality working by using only the ref?